libmscore/repeatlist.cpp

```diff
@@ -399,24 +399,29 @@ void RepeatList::unwind()
 
 Measure* RepeatList::jumpToStartRepeat(Measure* m)
       {
-      Measure* nm;
-      //
-      // go back to start of repeat or section break
-      // handle special case of end repeat (Measure m) has a section break
-      //
-      for (nm = m; nm && nm != _score->firstMeasure(); nm = nm->prevMeasure()) {
```

Comment:
if you notice, I have the removed the test for nm being non-null.  The reason is that the only reason why nm would be null is if there is something truely corrupt with the score such that the links between measures is somehow broken going backwards before reaching firstMeasure.  So that is probably better left as a test for corruption somewhere else in mscore...and if there was such a corruption in the score, it would instead be better to stop and report an error to user.  Maybe in some seperate test score for broken links function.